SEED-platform/seed
# 2331 refactor/rename column mapping profile

#### What's this PR do? 
Renames the ColumnMappingPreset model to ColumnMappingProfile. Updates all names (frontend and backend) from preset to profile as well.
Some files were not fully renamed, specifically any of the url/view files related to v2 and v2.1 apis.

Note, however this change _does_ require that use of these older endpoints now provide payloads that correspond to the new model (specifically, the attribute `preset_type` was changed to `profile_type`).

This is most likely not a commonly used api (and a relatively new field as well, within the last 3 months) so this is probably not a large issue, just something to be aware of.

#### How should this be manually tested?
Apply the migrations to update the schemas. Hard refresh your browser page to reset the cache

Column mapping profiles are used in a couple of places:
- `/app/#/accounts/<org id>/column_mappings/properties` (organization column mappings tab).
  - Verify "presets" isn't seen anywhere on screen.
  - You should see all of the orgs profiles in the dropdown
  - try: creating, updating, changing the name, and deleting profiles
  - see buildingsync test notes below
- `/app/#/properties/<id>` (property details page)
  - click the "Export BuildlingSync" action. The modal should show all of your orgs buildingsync profiles.
  - see the bsync test notes below
- `/app/#/data/mapping/5970` (data mapping page)
  - you should see the profiles populate the dropdown (according to the type of file you're importing)
  - create a new preset
  - see buildingsync test notes below

Also updated the service and view for downloading buildingsync. Test by going to inventory detail page then choosing the buildingsync export action.

There are some BuildingSync specific tests you should do as well. See the manual testing section in this PR: https://github.com/SEED-platform/seed/pull/2213

#### What are the relevant tickets?
#2331 
#2367
